src/fio/fioInfo.js

```diff
@@ -52,9 +52,7 @@ const defaultSettings: any = {
   fioRequestsTypes: FIO_REQUESTS_TYPES,
   balanceCurrencyCodes: {
     staked: 'FIO:STAKED',
-    available: 'FIO:AVAILABLE',
-    locked: 'FIO:LOCKED',
-    accrued: 'FIO:ACCRUED'
+    locked: 'FIO:LOCKED'
```

Comment:
Note: We kept staked because of the cost to refactor would be too high for today.